src/test/unit/controls/Graph/GraphAxes-spec.js

```diff
@@ -1997,4 +1997,21 @@ describe("Graph - Axes", () => {
             });
         });
     });
+    describe("Graph giv element for the label popup tooltip", () => {
+        describe("when graph is loaded", () => {
+            it("add div element for the popup tooltip", () => {
+                expect(
+                    fetchElementByClass(styles.labelPopupTooltip)
+                ).not.toBeNull();
+            });
+        });
+        describe("when graph is destroyed", () => {
+            it("remove div element for the popup tooltip", () => {
+                graph.destroy();
+                expect(
+                    fetchElementByClass(styles.labelPopupTooltip)
+                ).toBeNull();
+            });
+        });
```

Comment:
Nit 1: These describe blocks seem redundant; you can simply combine the `describe` and `it` statements.
Nit 2, grammar: add -> adds, remove -> removes

Reply:
nit: Probably, the upper describe block only looks redundant to me. You can remove that and keep the remaining `describe` and `it` blocks

you may use `should add` or `should remove` also

```js
        describe("when graph is loaded", () => {
            it("should add div element for the popup tooltip", () => {
                expect(
                    fetchElementByClass(styles.labelPopupTooltip)
                ).not.toBeNull();
            });
        });
        describe("when graph is destroyed", () => {
            it("should remove div element for the popup tooltip", () => {
                graph.destroy();
                expect(
                    fetchElementByClass(styles.labelPopupTooltip)
                ).toBeNull();
            });
        });
```